website/views/shared/profiles/stats.jade

```diff
@@ -23,13 +23,20 @@ table.table.table-striped
 
 // FIXME get translations working before can use this
 unless mobile
-  h4.stats-equipment(class=mobile?'item item-divider':'',ng-show='user.flags.itemsEnabled')=env.t('equipment')
+  h4(class=mobile?'item item-divider':'',ng-show='user.flags.itemsEnabled')=env.t('battleGear')
   table.table.table-striped(ng-show='user.flags.itemsEnabled')
     tr(ng-repeat='(k,v) in profile.items.gear.equipped', ng-init='piece=Content.gear.flat[v]', ng-show='piece')
       td
         strong {{piece.text()}}:&nbsp;
         span(ng-repeat='stat in ["str","con","per","int"]', ng-show='piece[stat]') {{piece[stat]}} {{stat.toUpperCase()}}&nbsp;
 
+unless mobile
+  h4(class=mobile?'item item-divider':'',ng-show='profile.preferences.costume && user._id != profile._id')=env.t('costume')
```

Comment:
I realize that the `item item-divider` classes aren't used, but I assumed they were left in for when the comment I mentioned in the first post got resolved. By my read, the `unless mobile` check is supposed to be temporary and is intended to be removed when whatever that comment is referring to is fixed. The comment was added over a year ago; maybe that issue has been fixed by now?